KRNumber2/KRNumber2/main.c

```diff
@@ -0,0 +1,98 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <locale.h>
+#include <stdbool.h>
+
+int scanOne(void);
+
+void bubbleSort(int arrayEvenNumbered[], int size) {
+	if (arrayEvenNumbered == NULL) {
+		return;
+	}
+	for (int i = 0; i < size; ++i) {
```

Comment:
```suggestion
	for (int i = 0; i < size - 1; ++i) {
```
А то последняя итерация цикла будет впустую